scripts/zones/Lebros_Cavern/mobs/Qiqirn_Mine.lua

```diff
@@ -0,0 +1,52 @@
+-----------------------------------
+-- Area: Lebros Cavern (Excavation Duty)
+-- MOB: Qiqirn Mine
+-----------------------------------
+local ID = require("scripts/zones/Lebros_Cavern/IDs")
+require("scripts/globals/allyassist")
+require("scripts/globals/status")
+-----------------------------------
+
+function onMobSpawn(mob)
+    mob:setMobMod(dsp.mobMod.NO_MOVE, 1)
+    mob:SetAutoAttackEnabled(false)
+end
+
+function onMobRoam(mob)
+    dsp.ally.startAssist(mob, dsp.ally.ASSIST_PLAYER);
+end;
+
+function onMobFight(mob,target)
+    local instance = mob:getInstance()
+    local players = instance:getChars();
+
+    for i,v in pairs(players) do
+        if mob:getLocalVar("Time") == 0 then
```

Comment:
I'll be the first to admit I don't do mobs, but you might be able to get away with setting the timers all at once, without needing to increment/check your time var every `onMobFight()`. If you can't do so in `onMobSpawn()`, another option in `onMobFight()` is to run a local var like `bomb_set`, and just check for `if not bomb_set then -- set the timers to explode the bomb end`

Reply:
Onmobfight only checks every 3 seconds. This way i have set will do a count every 1 second as i saw it in retail